documentation/overview.md

```diff
@@ -260,41 +270,38 @@ df.write.format("com.microsoft.cdm")
  .option("entityDefinitionContainer", "cdmmodels")
```

Comment:
You might want to remove entityDefinitionContainer and update entityDefinitionModelRoot -> cdmmodels/core